### ISSUE TITLE
Delete Fine Tune Model seems broken.

### ISSUE THREAD

**Issue by unixneo** [alexrudall/ruby-openai#183]
**Describe the bug**

Cannot delete a fine tune model.

**To Reproduce**

```
irb(main):001:0> fine_tuned_model="davinci:ft-personal-2023-01-24-03-13-55"
=> "davinci:ft-personal-2023-01-24-03-13-55"
irb(main):002:1*  Ruby::OpenAI.configure do |config|
irb(main):003:1*             config.access_token = ENV.fetch('OPENAI_API_KEY')
irb(main):004:0>         end
irb(main):005:0>         client = OpenAI::Client.new
=> #<OpenAI::Client:0x000000011553c7d8>
irb(main):006:0> client.finetunes.delete(fine_tuned_model:fine_tuned_model)
(irb):7:in `<main>': undefined method `delete' for #<OpenAI::Finetunes:0x0000000115497198> (NoMethodError)
```

**Expected behavior**

Per the docs:

```
 client.finetunes.delete(fine_tuned_model: fine_tuned_model)
```

But it does not work ..... 

However this does work:

```
 OpenAI::Client.delete(path: "/models/#{fine_tuned_model}")
=> {"id"=>"davinci:ft-personal-2023-01-24-03-13-55", "object"=>"model", "deleted"=>true}
```

**Comment by alexrudall** [comment]
Thanks @unixneo  - just released [3.1.0](https://rubygems.org/gems/ruby-openai/versions/3.1.0) which adds this!

**Comment by stefanoc** [comment]
@alexrudall actually it looks like at some point the `delete` method disappeared. This is the last version that I found:
```
    def delete(fine_tuned_model:)
      if fine_tuned_model.start_with?("ft-")
        raise ArgumentError, "Please give a fine_tuned_model name, not a fine-tune ID"
      end

      @client.delete(path: "/models/#{fine_tuned_model}")
    end
```
